gis/rsgislib/files/py311-rsgislib

```diff
@@ -0,0 +1,18 @@
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisapplycmd.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisbatchconvert2cog.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisbatchconvert2tif.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisbuildimglut.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgiscalcimgstats.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgischkgdalfile.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgiscopybandnames.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisfilehash.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisfilelut.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisimg2kmz.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisimg2webtiles.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisimginfo.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisproj.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisshepsegment.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgistranslate2cog.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgistranslate2tif.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisuserpassfile.py
+${frameworks_dir}/Python.framework/Versions/3.11/bin/rsgisvectools.py
```

Comment:
For Python, is anything different about each of these files, apart from the version?

If not, just use one file, with a placeholder for the version. (Something like `@@PYTHON_VER@@`.) Then use `reinplace` to fill that in, as appropriate.